pom.xml

```diff
@@ -162,7 +162,7 @@
         Consult https://chromedriver.chromium.org/downloads
         to find the right ChromeDriver version to use with the current browser version,
         -->
-        <chromedriver.version>113.0.5672.63</chromedriver.version>
+        <chromedriver.version>115.0.5790.102</chromedriver.version>
```

Comment:
The pr environment is using the equivalent version of chrome so this chromedriver version is now required.